Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.ts

```diff
@@ -302,23 +341,23 @@ class BocAutoCompleteReferenceValue //TODO RM-7715 - Make the TypeScript classes
 
   private ClearError(): void
   {
-    if (this._textbox.hasClass('error'))
+    if (this._textbox.classList.contains('error'))
     {
-      this._textbox.removeClass('error');
-      this._textbox.trigger('hideInformationPopUp');
+      this._textbox.classList.remove('error');
+      this.dispatchTextBoxEvent("hideInformationPopUp");
```

Comment:
Please use single quotes for strings in JS. Most (all?) JS code uses single quotes. File-consistency has last word on these matters.